Userland/Applications/PixelPaint/MainWidget.cpp

```diff
@@ -1347,6 +1347,16 @@ ErrorOr<void> MainWidget::create_image_from_clipboard()
     return {};
 }
 
+void MainWidget::refresh_crop_to_selection_menu_actions()
+{
+    m_crop_image_to_selection_action->set_enabled(true);
+    m_crop_layer_to_selection_action->set_enabled(true);
+    if (!current_image_editor() || current_image_editor()->image().selection().is_empty()) {
+        m_crop_image_to_selection_action->set_enabled(false);
+        m_crop_layer_to_selection_action->set_enabled(false);
+    }
```

Comment:
You could make this slightly simpler by doing something like:

```c++
auto enabled = current_image_editor() && !current_image_editor()->image().selection().is_empty();
m_crop_image_to_selection_action->set_enabled(enabled);
m_crop_layer_to_selection_action->set_enabled(enabled);
```

Reply:
done